frontend/src/component/common/Search/SearchSuggestions/SearchSuggestions.tsx

```diff
@@ -72,9 +72,9 @@ export const SearchSuggestions: VFC<SearchSuggestionsProps> = ({
             return {
                 name: column.filterName,
                 header: column.Header ?? column.filterName,
-                options: [...new Set(filterOptions)].sort((a, b) =>
-                    a.localeCompare(b)
-                ),
+                options: [...new Set(filterOptions)]
+                    .filter((it: unknown) => it)
```

Comment:
filter out empty suggestions

Reply:
Is this the same as `.filter(Boolean)`? Maybe that would be cleaner.